### ISSUE TITLE
Easee EV Charger: Action command Stop doesnt work ? 

### ISSUE THREAD

**Issue by t112013** [fondberg/easee_hass#273]
### The problem

Hello. i have just installed the integration and everything seems fine, i have two charges and servce start does work but stop doesnt ? pause does. 
Is it some limitations ? 

### Version of Easee integration having the issue?

0.9.48

### Version of Home Assistant Core having the issue?

2023.2.3

**Comment by olalid** [comment]
Hi,
You have likely misunderstood what the stop commands purpose is.
See https://developer.easee.cloud/docs/settings
I.e. it does not do anything if your charger is not set up with authorisation with RFID card.

**Comment by t112013** [comment]
Thanks for great and fast answer. hmmmm  i see i see. 
Best just to use toggle for simple start/stop ? 
One quick question, anyhow to resert the session ? i saw that toggle, stopping and then starting will continue the meter.

**Comment by olalid** [comment]
I personally prefer to use the dynamic circuit limit instead of start/stop/pause/resume. It just always works independent of what mode etc you are in.
Set it to something less than 6A and the charging will stop, and set it to 6A and more and charging will resume.
There are a lot of other support issues here that you can read through to get some ideas.

I am not sure I understand you correctly, do you want to reset the session energy sensor? Not possible as far as I know, the cable has to be unplugged for it to restart from 0.

**Comment by t112013** [comment]
thanks a lot, yes your last answe was right on.
Okidoki regarding the less than 6A, that would stop both cars right ? 
I will try to read arround here. thanks a lot :)

**Comment by olalid** [comment]
If you have 2 chargers connected to the same circuit, setting the dynamic circuit limit to less than 6A will stop both cars from charging, yes.

**Comment by t112013** [comment]
I have firured out that you can also use the set the chargers maks value too. that one will actually stay that value until chnged ( dynamic value gets reset on replugg)

I have also learned that you can ues the 'is enabled' switch to turn the charger on and off.  nice touch to keep a charger that is monuted ouside and accesible to anyone from beein used by someoneelse 

Thank for your help :)

**Comment by olalid** [comment]
Well, it is only charger dynamic limit that is reset on re-plug, not circuit dynamic limit.

There are in any case a lot of different ways to accomplish the same thing.
Note that the non-dynamic limits will write to flash in the unit, so if you make a lot of changes it might wear the flash out.

**Comment by t112013** [comment]
right, i have set it one time only, the dynamic value will command for both cargers when charging at the same time. 
Does the 'is enabled' switch write to the flash also ?

**Comment by olalid** [comment]
Sorry, I do not know if the is_enabled switch it writes to flash.
It ends up calling this API: https://developer.easee.cloud/reference/post_api-chargers-id-settings
That is all I can tell you...

All I can say is that limit settings that are not called "dynamic", like charger max limit, or offline limit will write to flash.
So if you are doing things like load balancing where you would update it like once a minute or so, you should not use those.

**Comment by t112013** [comment]
no iam just turnning on and off the outside charger (i have one on the driveway outside, where everybody can just plug kind of)  on demand when my car also updates its value to connected and gps is home then turning it off when i unplugg the cable. 
so two changes  per charging. dont know if that will breake the flash, i will try to contact support.
